### ISSUE TITLE
Figure with alt text fails to render

### ISSUE THREAD

**Issue by bluewin4** [quarto-dev/quarto-cli#8207]
### Bug description

Was using the ACM template to generate a file for writing my paper and had some trouble compiling it into a pdf. I narrowed the problem down to this:
```
![1907 Franklin Model D roadster. Photograph by Harris \& Ewing, Inc. [Public domain], via Wikimedia Commons. (<https://goo.gl/VLCRBB>).](sample-franklin.png){#fig-dresses fig-alt="A woman and a girl in white dresses sit in an open car."}
```
As long as there is the `{fig-alt="A woman and a girl in white dresses sit in an open car."}` the pdf fails to render, upon removal it works just fine.

 The only thing I saw related to this was #4142 but it seems to be a separate issue.

### Steps to reproduce

Run 
```
quarto use template quarto-journals/acm
```
Then attempt to render

### Expected behavior

Produces a template pdf

### Actual behavior

Fails to produce a pdf

### Your environment

VSCode on Ubuntu

### Quarto check output

Quarto 1.4.538
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.538
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: v2024.01
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /home/jack/.TinyTeX/bin/x86_64-linux
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.12
      Path: /bin/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

      There is an unactivated Python environment in obs. Did you forget to activate it?

[✓] Checking R installation...........OK
      Version: 4.3.2
      Path: /usr/lib/R
      LibPaths:
        - /home/jack/R/x86_64-pc-linux-gnu-library/4.3
        - /usr/local/lib/R/site-library
        - /usr/lib/R/site-library
        - /usr/lib/R/library
      knitr: 1.44
      rmarkdown: 2.25

[✓] Checking Knitr engine render......OK

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Comment by mcanouil** [comment]
The following compiles properly:

````qmd
---
title: "Quarto Playground"
format: pdf
---

This is a playground for Quarto.

![1907 Franklin Model D roadster. Photograph by Harris \& Ewing, Inc. [Public domain], via Wikimedia Commons. (<https://goo.gl/VLCRBB>).](https://placehold.co/600x400.png){#fig-dresses fig-alt="A woman and a girl in white dresses sit in an open car."}
````

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/44826dea-a87d-4086-a256-00cf33a2dbb5">

**Comment by mcanouil** [comment]
It seems to be an issue with the template extension which causes conflict, *i.e.*, an invalid figure float is created:

```latex
\begin{figure}

\centering{

\begin{figure}
{\centering \includegraphics{sample-franklin.png}}
\caption{}
\Description{A woman and a girl in white dresses sit in an open car.}
\end{figure}

}
```

While the default LaTeX template produces:

```latex
\begin{figure}

\centering{

\includegraphics{index_files/mediabag/600x400.png}

}

\caption{\label{fig-dresses}1907 Franklin Model D roadster. Photograph
by Harris \& Ewing, Inc.~{[}Public domain{]}, via Wikimedia Commons.
(\url{https://goo.gl/VLCRBB}).}

\end{figure}%
```

**Comment by cderv** [comment]
I opened 
* https://github.com/quarto-journals/acm/issues/36

to follow up on this

**Comment by bluewin4** [comment]
Thanks so much for looking into this!
